knexfile.js

```diff
@@ -12,13 +12,22 @@ module.exports = {
       port : 3306,
       user : process.env.MYSQL_USERNAME,
       password : process.env.MYSQL_PASSWORD,
-      database : process.env.MYSQL_DATABASE
+      database : process.env.MYSQL_DATABASE,
+      socketPath: '/var/run/mysqld/mysqld.sock'
```

Comment:
don't hardcode connection params, even for the local development, use env variables

Reply:
replaced with env variables